modules/installation-about-disconnected.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_disconnected/installing-disconnected-preparations.adoc
+
+[id="installation-about-disconnected_{context}"]
+= About disconnected installations
+
+In {product-title} {version}, you can perform an installation that does not
+require an active connection to the internet to obtain software components. You
+complete a disconnected installation on only infrastructure that you provision,
+not infrastructure that the installer provisions, so your platform selection is
+limited.
+// maybe point out that you can follow the bare metal installation rules on supported hardware and link to the matrix
+
+If you choose to perform a disconnected installation on a cloud platform, you
+still require access to its cloud APIs and will still require internet access
+to use your cluster. Depending on your network, you might require less internet
+access for an installation on bare metal hardware or on VMware vSphere.
+
+To complete a disconnected installation, you must create a repository that
+mirrors the contents of the {product-title} registry and contains the
+installation media. You can create this mirror on a bastion host, which can
+access both the internet and your closed network, or by using other methods
+that meet your restrictions.
+
+[IMPORTANT]
+====
+Because disconnected installations are always on user-provisioned infrastructure,
+always complete a standard user-provisioned infrastructure installation before
+you attempt a disconnected installation.
+====
+
+[id="installation-disconnected-limits{context}"]
+== Additional limits
+
+Disconnected clusters have the following additional limitations and restrictions:
+
+* Cluster settings always displays errors because the cluster cannot determine
```

Comment:
"Cluster settings" is not very specific.  Maybe "The ClusterVersion status will include `[Unable to retrieve available updates][0]` unless you update its [`spec.upstream`][1]."?  And hmm, maybe we want to patch the ClusterVersion object to empty that property, since it's optional.  Not sure if the best place to patch it is clobbering [this][2] in a multi-part install or to patch the object post-install.  Probably more reliable to patch post-install, since then you wouldn't be impacted in the unlikely event that the installer renames its generated manifest file.  But then there will be a window of time before the patch when you will still see the error.  Or maybe clearing the `upstream` field will [still give you errors][3].  @abhinavdahiya, thoughts about the best-practices config for "I don't want to upgrade this cluster ever"?

[0]: https://github.com/openshift/cluster-version-operator/blob/0725bd53c732c8e3a490d1080f03edbe7e579b3a/pkg/cvo/availableupdates.go#L165
[1]: https://github.com/openshift/api/blob/cdbe64fb0c9184ffe47d24d0ea490705a92d9fd8/config/v1/types_cluster_version.go#L54-L58
[2]: https://github.com/openshift/installer/blob/d9a9648cf2330d467cca9f2988846d031464125e/data/data/manifests/bootkube/cvo-overrides.yaml.template#L7
[3]: https://github.com/openshift/cluster-version-operator/blob/0725bd53c732c8e3a490d1080f03edbe7e579b3a/pkg/cvo/availableupdates.go#L192

Reply:
Can you tell me more about this patch? Do we want users to patch this field? Can I just tell them that there's an error?

Reply:
Telling them ClusterVersion will report an error (more specific than what you have now) is fine, but if it was my cluster I'd patch to remove it ;).

Reply:
@wking, do you have information about patching `spec.upstream` somewhere?

Reply:
Should be:

```console
$ oc patch clusterversion version --type json -p '{"op": "remove", "path": "/spec/upstream"}'
```
But let me sanity check, and I'll report back.

Reply:
So for "I want to watch for updates in a different location":

```console
$ oc patch clusterversion version --type json -p '[{"op": "add", "path": "/spec/upstream", "value": "https://example.com/your/new/location"}]'
```

And for "I don't want to check for upgrades, I'll either never upgrade (!) or manually upgrade (e.g. via `oc adm upgrade ...`)":

```console
$ oc patch clusterversion version --type json -p '[{"op": "remove", "path": "/spec/channel"}, {"op": "remove", "path": "/spec/upstream"}]'
```

in which case ClusterVersion will have a `status.conditions` entry like:

```yaml
  - lastTransitionTime: "2019-09-21T22:36:52Z"
    message: The update channel has not been configured.
    reason: NoChannel
    status: "False"
    type: RetrievedUpdates
```

which seems appropriate.

Reply:
Although it looks like we may want to file something around NoChannel for a console update, because `/settings/cluster` currently puts up a moderately frightening stop sign and doesn't expose the reason or message (unless I'm just missing something about the UI):

![Screenshot from 2019-09-21 16-21-12](https://user-images.githubusercontent.com/209920/65380249-eb6f1000-dc8b-11e9-83db-710174f4428a.png)

Reply:
Did you file a bug about that console issue? I think that more text would certainly help there.

I'm making a [follow-up issue ](https://jira.coreos.com/browse/OSDOCS-695)to describe patching the CVO. I don't see any modifications to it in our current doc set, and with our docs workflow, it will be easier to get reviews on the proposed change on a separate PR.